src/actions/actionDuplicateSelection.ts

```diff
@@ -11,9 +11,10 @@ export const actionDuplicateSelection = register({
       elements: elements.reduce(
         (acc: Array<ExcalidrawElement>, element: ExcalidrawElement) => {
           if (appState.selectedElementIds[element.id]) {
```

Comment:
👍🏽

Reply:
That's thumbs up I hope :D.

My system sucks:

![image](https://user-images.githubusercontent.com/5153846/76874109-fab3cc80-686e-11ea-9851-dcc7ad8195e1.png)

Reply:
Yea, it is!!
Nice work on this PR :)